packaging/hudi-cli-bundle/pom.xml

```diff
@@ -239,5 +241,11 @@
       <artifactId>httpclient</artifactId>
       <version>${http.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-configuration2</artifactId>
```

Comment:
Why do we need to declare if it is included above? Is this dependency in provided scope in parent pom?

Reply:
This is missing based on @rahil-c 's testing.  Without this, hudi-cli-bundle will fail with Spark 3.2 + Hadoop 3.